core/templates/dev/head/pages/exploration-editor-page/statistics-tab/templates/playthrough-modal.template.html

```diff
@@ -3,27 +3,26 @@ <h3 class="modal-title">Sample Playthrough</h3>
 </div>
 
 <div class="modal-body" style="padding-bottom: 8rem; margin-left:0.5rem; margin-right: 0.5rem;">
-  <div ng-if="maxHidden != 0" class="row oppia-pt2">
-    <button type="button" name="button" ng-click="showRemainingActions(playthroughIndex)" class="oppia-issues-btn-arrow" id="arrowDiv" style="display: block;"><i class="fas fa-arrow-up"></i></button>
+  <div ng-if="canExpandActions()" class="row oppia-pt2">
+    <button ng-show="canExpandActions()" type="button" name="button" ng-click="expandActionsToRender()" class="oppia-issues-btn-arrow" id="arrowDiv" style="display: block;"><i class="fas fa-arrow-up"></i></button>
```

Comment:
Why do you have an ng-show inside an ng-if with the same condition?

Reply:
Removed.